src/parser/test/test.ml

```diff
@@ -3401,6 +3468,148 @@ let%expect_test _ =
               \nCode blocks should be indented at the opening `{`.")))))
          (warnings ()))
         |}]
+
+    (** {3 Metadata tags}
+
+        Code blocks have a metadata field, with bindings and simple tags. *)
+
+    let lots_of_tags =
+      test
+        {|{@ocaml env=f1 version=4.06 "tag with several words" "binding with"=singleword also="other case" "everything has"="multiple words" [foo]}|};
+      [%expect
+        {|
+        ((output
+          (((f.ml (1 0) (1 137))
+            (code_block
+             (((f.ml (1 2) (1 7)) ocaml)
+              ((binding ((f.ml (1 8) (1 11)) env) ((f.ml (1 12) (1 14)) f1))
+               (binding ((f.ml (1 15) (1 22)) version) ((f.ml (1 23) (1 27)) 4.06))
+               (tag ((f.ml (1 29) (1 51)) "tag with several words"))
```

Comment:
I think the location of quoted tags/bindings should include the quotes. Otherwise it's hard for programs that want to change the tags by modifying what is at this location...

Reply:
Yes, that's fair.